src/bookshelf/schema/deploy/utility_functions.sql

```diff
@@ -0,0 +1,71 @@
+-- Deploy bookshelf:utility_functions to pg
```

Comment:
Do we need this?  Most aren't called in the code or are called by functions which aren't called according to xref.